### PR TITLE
Restore multiplicities

### PR DESCRIPTION
**Pull request by jmduarte**
Related PR: https://github.com/p2l1pfp/FastPUPPI/pull/40

### CODE REVIEW

File: L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc [hunk 1]
```diff
@@ -224,7 +235,7 @@ L1TPFProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
 
     // Then go do the multiplicities
     // FIXME: recover
-    /*
+
```
Comment (gpetruc) [inline L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc:244]: Remove the also the FIXME above

**Review by gpetruc — CHANGES_REQUESTED** [review]

File: L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc (hunk 1 above)
Comment (jmduarte) [reply L1Trigger/Phase2L1ParticleFlow/plugins/L1TPFProducer.cc:244]: Removed

**Comment by jmduarte** [comment]
@gpetruc I updated the PR. Let me know if I should rebase and make it to another branch as well.

File: L1Trigger/Phase2L1ParticleFlow/python/l1ParticleFlow_cff.py [hunk 2]
```diff
@@ -131,24 +131,39 @@
...
+        etaExtra = cms.double(0.3),
+        phiExtra = cms.double(0.0)
+    ),
+))
 
 l1ParticleFlow_pf_hgcal = cms.Sequence(
     pfTracksFromL1TracksHGCal +   
-    l1pfProducerHGCal
+    l1pfProducerHGCal + 
+    l1pfProducerHGCalNoTK
```
Comment (gpetruc) [inline L1Trigger/Phase2L1ParticleFlow/python/l1ParticleFlow_cff.py:166]: did you make some check that this makes no difference in the PF & Puppi output

**Comment by gpetruc** [comment]
closing, now integrated in the 10_6_1p2 branch

**Pull request closed without merging**